odl/util/utility.py

```diff
@@ -1607,6 +1608,28 @@ def unique(seq):
         return unique_values
 
 
+def nextpow2(n):
+    """
+    Compute the integer which is a power of two.
```

Comment:
On first line.
An maybe: ~"Return the next integer power of 2."~
Actually, after the change, it's more like "Return the exponent of the next integer power of 2."